go/worker/event/server.go

```diff
@@ -175,7 +175,7 @@ func PprofCpuStop(w http.ResponseWriter, _ *http.Request) {
 }
 
 // Writes final stats and any buffered CPU profile to disk.
-func WriteFinalStats(pidPath string, server cleanable) {
+func WriteFinalStats() {
```

Comment:
I think this change is already in.  You probably need to merge main to your branch and re-push to the PR so I only see the real changes.